packages/executor/HiveTask/HiveTaskBuilder.ts

```diff
@@ -0,0 +1,61 @@
+import {IHive} from "@moonset/model";
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import {FileAsset, StringAsset} from "../lib/cdk/asset";
+import * as sfnTasks from "@aws-cdk/aws-stepfunctions-tasks";
+import {MoonsetConstants as MC} from "../lib/constants";
+
+const HiveSQLNotExistError = "Either sqlFile or sql must exist for hive.";
```

Comment:
This code is out dated. Please check plugin-platform-emr, and refactor the code into new code base.